server/routes/api.py

```diff
@@ -55,5 +59,23 @@ def post(author_id):
         db.session.add(post)
         db.session.commit()
         return Response(status=httpStatus.OK)
+    
+@bp.route('/hello_world')
+def hello_world():
+    return make_response(jsonify(message='Hello World')), 200
+
+@bp.route('/verify_login')
+def verify_login():
+    # get token from authorization header
+    authorization_header = request.headers['Authorization']
+    token = authorization_header.replace("Bearer ", "")
 
+    try:
+        decoded_token = auth.verify_id_token(token)
+        uid = decoded_token['uid']
+        print("UID", uid)
```

Comment:
Are you going to add the Author database creation on a different PR?

Reply:
I was planning on leaving this like a sample endpoint but I might as well just add it here

Reply:
I also added the author to the database and created an api_support.py file where the functions needed by api.py are located. I noticed that api.py was not modularized which is why I created that file for the functions used in my part

Reply:
I also added the author to the database and created an api_support.py file where the functions needed by api.py are located. I noticed that api.py was not modularized which is why I created that file for the functions used in my part